student.rb

```diff
@@ -0,0 +1,12 @@
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] You could even use a string for `parent_permission` instead of a symbol
```suggestion
  def initialize(age, specialization, name = 'Unknown', parent_permission = true)
```